src/DotNet.Testcontainers/Containers/ResourceReaper.cs

```diff
@@ -80,10 +96,11 @@ private ResourceReaper(Guid sessionId, string ryukImage)
     /// <summary>
     /// Starts and returns the default <see cref="ResourceReaper" /> instance.
     /// </summary>
+    /// <param name="dockerEndpoint">The Docker API endpoint.</param>
     /// <param name="ct">The cancellation token to cancel the <see cref="ResourceReaper" /> initialization.</param>
     /// <returns>Task that completes when the <see cref="ResourceReaper" /> has been started.</returns>
```

Comment:
`/// <remarks>If <paramref name="endpoint" /> is null, the resource reaper will fallback to the default Docker API endpoint.</remarks>` (append to the other methods as well)